src/components/content/catalog/services/history/ServiceTemplateRequested.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Alert, Skeleton } from 'antd';
+import React from 'react';
+import DisplayOclData from '../../../common/ocl/DisplayOclData';
+import useRequestedServiceTemplateQuery from './useRequestedServiceTemplateQuery';
+
+function ServiceTemplateRequested({ requestId }: { requestId: string }): React.JSX.Element {
+    const requestedServiceTemplateQuery = useRequestedServiceTemplateQuery(requestId);
+
+    return (
+        <div>
+            {requestedServiceTemplateQuery.isLoading ? (
+                <Skeleton active />
+            ) : requestedServiceTemplateQuery.isSuccess ? (
+                <DisplayOclData ocl={requestedServiceTemplateQuery.data} />
+            ) : requestedServiceTemplateQuery.isError ? (
+                <Alert
```

Comment:
we must handle the error fully. For known errors, we must handle it like we have do it in all other components.